### PR TITLE
Matomo

### PR DESCRIPTION
**Pull request by vroland**
Work in Progress - do not merge.

### CODE REVIEW

**Comment by inyono** [comment]
I rebased this PR to the current master and updated the refs of the modules (after merging the dependent PRs)

File: variables.tf [hunk 1]
```diff
@@ -37,8 +37,18 @@ variable "athene2_php_smtp_password" {
...
+  default     = "false"
+}
+
+variable "athene2_enable_tracking_google_analytics" {
+  description = "Flag whether to activate Google Analytics tracking or not -> usually only set to true in production"
+  default     = "false"
+}
+
+variable "athene2_enable_tracking_matomo" {
+  description = "Flag whether to activate Matomo tracking or not -> usually only set to true in production"
```
Comment (inyono) [inline variables.tf]: Don't need to be variables. You can move that as locals to `serlo-org.tf`

File: matomo.tf [hunk 2]
```diff
@@ -0,0 +1,38 @@
...
+
+  app_replicas  = 1
+  image_tag     = local.matomo.matomo_image_tag
+  namespace     = kubernetes_namespace.matomo_namespace.metadata.0.name
+  database_host = module.gcloud_mysql.database_private_ip_address
+  database_user = "serlo"
+  #var.matomo_mysql_password
+  database_password    = var.athene2_database_password_default
+  database_name        = "matomo"
+  persistent_disk_name = "matomo-storage-${local.project}"
```
Comment (inyono) [inline matomo.tf:18]: Do we need that as a variable? Can't the matomo module handle that itself?

Comment (inyono) [reply matomo.tf:18]: nvm, you didn't add that disk yet

**Comment by inyono** [comment]
What's the current state of this PR? Are there any blockers?

**Comment by vroland** [comment]
The variables are now converted to locals, as you suggested. No further blockers as far as I know.

**Comment by inyono** [comment]
Nice. The PVC was still in the default namespace, apart from that it looks good.
- Can you please send me the super user credentials via Slack?
- Can you please document anything special you needed to do to configure in Matomo setup / after the installation wizard?

**Review by inyono — APPROVED** [review]

**Pull request merged**